src/algorithms/calorimetry/CalorimeterHitDigi.cc

```diff
@@ -1,5 +1,6 @@
 // SPDX-License-Identifier: LGPL-3.0-or-later
-// Copyright (C) 2022 Chao Peng, Wouter Deconinck, Sylvester Joosten, Barak Schmookler, David Lawrence
+// Copyright (C) 2022 Chao Peng, Wouter Deconinck, Sylvester Joosten, Barak Schmookler, David
+// Lawrence
```

Comment:
Not great. Can we leave comments alone?

Reply:
I don't mind reflowing comments. If we keep adding people to the copyright line, then we have to. As long as it is smart about reflowing. It seems to be at least somewhat smart since it didn't make this:
```
// SPDX-License-Identifier: LGPL-3.0-or-later Copyright (C) 2022 Chao Peng, Wouter
// Deconinck, Sylvester Joosten, Barak Schmookler, David Lawrence
```

Reply:
Disabled comment reflow in fa0a0800.